pkg/auth/rhsso_authz_handler_test.go

```diff
@@ -93,6 +97,129 @@ var _ = Describe("Authz email domain", func() {
 	}
 })
 
+var _ = Describe("autz_ownership_api", func() {
+	var (
+		db      *gorm.DB
+		dbName  string
+		ctx     context.Context
+		handler Authorizer
+	)
+	BeforeEach(func() {
+		db, dbName = common.PrepareTestDB()
+		ctx = context.Background()
+	})
+
+	AfterEach(func() {
+		common.DeleteTestDB(db, dbName)
+	})
+
+	Context("authz RHSSO ownership", func() {
+		Context("tenancy disabled", func() {
+			BeforeEach(func() {
+				cfg := &Config{AuthType: TypeRHSSO, EnableOrgTenancy: false}
+				handler = NewAuthzHandler(cfg, nil, logrus.New(), db)
+			})
+			It("admin user - no conditions", func() {
+				payload := &ocm.AuthPayload{}
+				payload.Role = ocm.AdminRole
+				ctx = context.WithValue(ctx, restapi.AuthKey, payload)
+
+				sql := db.ToSQL(func(tx *gorm.DB) *gorm.DB {
+					return handler.OwnedBy(ctx, tx).First(&models.Cluster{})
+				})
+				Expect(sql).NotTo(ContainSubstring("id = "))
+				Expect(sql).NotTo(ContainSubstring("user_name"))
+				Expect(sql).NotTo(ContainSubstring("org_id"))
```

Comment:
We have a connection to the db. Why not actually create the rows and ensure you get the right ones back rather than doing string comparisons on the sql?

Reply:
Replaced by real test data and assertions on the retrieved records